### DISCUSSION TITLE
New 1.2.0 GUI for windows no longer auto refreshing sense 1.2.0

### DISCUSSION THREAD

**Discussion by ZielReaver** [Chia-Network/chia-blockchain#7217] [Support]
Hello I noticed this morning when I woke up all three of my systems is not updating my completed plots. I had to hit the plot refresh button on all three system to get them to update there numbers.

**Comment by loppefaaret** [comment]
We might have some display issues with the new plots tab - but it should work underneath

**Reply by 7by24** [reply]
same issue here.

**Comment by CarlssonJD** [comment]
Same issue here. I need to press Keys section on the GUI and press my private key and then everything refreshes and the plots are showing. I need to do this each time I make a new plot.

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
